tests/pycompare/generate-pycompare.R

```diff
@@ -48,3 +48,32 @@ py_kap <- list(
 )
 saveRDS(py_kap, "tests/pycompare/py-kap")
 
+# Kappa weighted
+
+# We can't send ordered factors into sklean (!), so this was effectively
+# generating random numbers at first. Tweaking to use as.integer of the factors
+# for true comparison.
+truth <- factor(
+  hpc_cv$pred, levels = levels(hpc_cv$pred), ordered = TRUE
+)
+estimate <- factor(
+  hpc_cv$obs, levels = levels(truth), ordered = TRUE
+)
+truth <- as.integer(truth)
+estimate <- as.integer(estimate)
+
+py_kap_weighted_linear <- list(
+  binary = skmetrics$cohen_kappa_score(two_class_example$truth, two_class_example$predicted, weights = "linear"),
+  multiclass = skmetrics$cohen_kappa_score(truth, estimate, weights = "linear")
+)
+saveRDS(py_kap_weighted_linear,
+        "tests/pycompare/py-kap-weighted-linear",
+        version = 2)
```

Comment:
Does the version matter?

Reply:
It mattered for Travis. The old-version-of-R check failed when I didn't include the version. It doesn't matter to *me*, per se, but I wanted to clear the red.